Documentation/Signoffs/Power_System_Camera.md

```diff
@@ -1,37 +1,125 @@
 # Power System for Camera System
 ## Functionality
+The primary functionality for the camera power system is to reliably meet the power requirements for the camera hardware system's components. The system will supply 5 V to the camera system's Raspberry Pi 5 via a USB-C connection and 5 V to the Raspberry Pi HQ camera. The system will supply 5 V to both servo motors +5V DC and GND terminals.
 
 ## Constraints
 | No.| Contraint | Origin |
 | -- | --------- |--------|
-|  1 |           |        |              
-|  2 |           |        |                          
-|  3 |           |        |              
+|  1 | The system shall comply with ASTM B258-18 [2]. | Standards and Safety |
+|  2 | The system shall be weather resitant to an IP67 rating [1].  | Standards and Safety |
+|  3 | The system shall be able to supply enough power to the camera system for its full functionality | Design Team | 
+|  4 | The system shall not be impacted by back EMF from the servo motors | Design Team | 
+|  5 | The system shall contain a manual switch to individually cut off power supplied | Standards and Safety | 
 
```

Comment:
This system (as mentioned later) plugs into mains power so it would be really really good to include constraints regarding (a) shock protection and (b) overcurrent protection.

Reply:
Constraint added for shock and overcurrent protection.